conda-arm64-requirements.txt

```diff
@@ -1,3 +1,5 @@
 # with moving to arm64 for the new M1/M2 macs some packages are not yet compatible via pip and require
 # conda which supports different architecture environments on the same machine
 lightgbm>=3.0
+protobuf>=3.20.3, <4
```

Comment:
need to explain why it is limited (also pyyaml)

Reply:
f30ae9f